src/query_parser/v2/parser.y

```diff
@@ -982,6 +1057,21 @@ vector_attribute_list(A) ::= vector_attribute(B). {
   A.needResolve = array_append(A.needResolve, B.needResolve);
 }
 
+/*** Vector range queries ***/
+expr(A) ::= modifier(B) COLON LSQB vector_range_command(C) RSQB. {
+    C->vn.vq->property = rm_strndup(B.s, B.len);
+    A = C;
+}
+
+vector_range ::= VECTOR_RANGE_S .
+vector_range ::= VECTOR_RANGE_T .
+
+vector_range_command(A) ::= vector_range param_num(B) ATTRIBUTE(C). {
```

Comment:
What does ATTRIBUTE mean in this context?

Reply:
This is token of query param, such as `$blob` in `@v:[vector_range 0.1 $blob]`

Reply:
After talking with @oshadmi, I think we should opt for -
`vector_range_command(A) ::= term(B) param_any(C) param_any(C)`